CommonMark/Formatters/HtmlFormatterSlim.cs

```diff
@@ -30,6 +31,8 @@ internal static class HtmlFormatterSlim
             true,  true,  true,  true,  true,  true,  true,  true,  true,  true,  true,  false, false, false, false, false
         };
 
+        private static readonly Dictionary<string, uint> GitHubHeaderIdCounts = new Dictionary<string, uint>();
```

Comment:
Please suggest an actual alternative, rather than what *not* to do.  If I don't use the dictionary to store ID counts, what do I use?